Vagrantfile

```diff
@@ -350,6 +350,9 @@ Vagrant.configure('2') do |config|
     # Reference: https://github.com/chef/vagrant-omnibus/issues/90#issuecomment-51816397
     cfg.vm.guest = :windows
     default_omnibus config
+    # config below prevents the installation of latest Chef on the box.
+    # Reference: https://github.com/chef/vagrant-omnibus/issues/129
```

Comment:
Minor, issue 129 was commented as a duplicate of issue 118.  Since most of the useful comments and workarounds are on 118, I would prefer to reference that issue instead.

Reply:
c56339bcc3223c7413cf699102abb7237d3f8049